.eslintrc

```diff
@@ -1267,7 +1267,7 @@ rules:
 
   # Sort import declarations within module.
   sort-imports:
-    - 1
+    - 0
     -
       ignoreCase: false
       ignoreMemberSort: false
```

Comment:
Can we get rid of the config we no longer need here, as well? (+5 lines below that couldn't be selected for commenting)

Reply:
We have several others that are turned off but have the options. I originally did this so if the defaults changed in an ESLint upgrade, the rules wouldn't automatically change their defaults. Maaaaaybe something better for its own PR to  remove them for all turned off rules, so that this is just sort-imports.

Reply:
Hmm looking at some of the other ones I'm inclined to just leave it. If we turn these on again we don't need to figure out the format of the options.